File: Day 012/classwork/classwork.py

# number = int(input("please enter number: "))

# result = 0

# while number >= 0:
#     result = result + number
#     number = number - 1

# print(result)



#მომხმარებელს შემოატანინეთ ორი რიცხვი, ხოლო შემოტანის შემდეგ for ციკლის გამოყენებით გამოუტანეთ რიცხვები, ორი შემოტანილი რიცხვის დიაპაზონში
#მაგალითად, თუ მომხმარებელმა შემოიტანა 4 და 8 გამოუტანეთ რიცხვები 4,5,6,7,8

user_input = int(input("Please enter first number: "))
user_input2 = int(input("Please enter Second number: "))

for char in range(user_input, user_input2 + 1):
    print(char)

File: Day 137/classwork/classwork.js
import './App.css';
import { useState } from 'react';

export default function App() {
  const [products, setProducts] = useState([]);

  const handleSubmit = (e) => {
    e.preventDefault();

    const product = {
      name: e.target.name.value,
      price: e.target.price.value,
    };

    setProducts([...products, product]);

    e.target.name.value = '';
    e.target.price.value = '';
  };

  return (
    <main>
      <form onSubmit={handleSubmit}>
        <input
          type="text"
          placeholder="Please enter product name"
          name="name"
          required
        />
        <input
          type="number"
          placeholder="Please enter product price"
          name="price"
          required
        />
        <button type="submit">Submit</button>
      </form>

      <ul>
        {products.map((product, index) => (
          <li key={index}>
            {product.name} - ${product.price}
          </li>
        ))}
      </ul>
    </main>
  );
}
